### DISCUSSION TITLE
Formatting blog posts

### DISCUSSION THREAD

**Discussion by Pixelest** [xenocrat/chyrp-lite#181] [Q&A] [answered]
Hi I'm looking for some help on formatting blog posts. It seems as though when I enter things like new lines and code markdown it just displays everything as a long running paragraph although it is different in the post window. 
I am attaching two files that will hopefully show what I am talking about. 

![image_2023-01-15_141313040](https://user-images.githubusercontent.com/2861198/212562011-fb843eb5-cca6-4701-9090-16d7f5317de5.png)
![image](https://user-images.githubusercontent.com/2861198/212562024-84415d05-2a17-43bf-8f36-c383483c8b8c.png)

**Comment by nexoft-xyz** [comment]
The following doesn't resolve the main issue, but I had the same issue until I discovered that adding an indentation (tab) instead to each line gave a better result than what is suggested by the markdown insight. Adding indentation is not listed in the Markdown as well and seems to be a hidden feature?


So doing this :
![Screenshot_20230115_093749](https://user-images.githubusercontent.com/33012133/212566109-1568c830-bd61-413b-bd6e-efd596ff7d74.png)

Gives this : 
![Screenshot_20230115_093639](https://user-images.githubusercontent.com/33012133/212566080-096ceeac-c8dd-4fab-84bf-9a409bfd8761.png)

I don't know if it is intended, and if there are any hidden features to format the text. I never found the time to raise this before. ( I would like to contribute but unfortunately I have neither the time or the knowledge for now. )

**Reply by Pixelest** [reply]
Thanks this works as a temporary solution for now I hope. Wont mark this as answered just yet unless the creator has some input.

**Comment by xenocrat** [comment, marked as answer]
Hello there,

Content formatting in Chyrp Lite follows HTML [whitespace handling](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Whitespace) rules. In a nutshell, any number of consecutive spaces, tabs, or new lines in the source will usually be truncated to a single space when rendered for display. If you want a hard line break at a specific location, you should enter [`<br>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br). With Markdown formatting enabled in Chyrp Lite's settings, you can also create a paragraph break by leaving an entirely blank line in the source...

... like this. This and more rules can be found [here](https://commonmark.org/help/). Adding a tab or 4 consecutive spaces at the start of a line will render it as [`<pre>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre) - a pre-formatted block of text. If you disable Markdown formatting in the settings, you'll need to rely on HTML tags to structure your content: most commonly [`<p>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p), [`<br>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br), and [`<pre>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre) .

**Reply by Pixelest** [reply]
This is what I am looking for thank you!
